README.md

```diff
@@ -4,56 +4,63 @@ Website: http://forks-insight.com or http://128.2.112.151
 
 Redesign for https://github.com/shuiblue/INFOX
 
-How to run:
-1. Ramp up the environment according to environment.yaml (you can load by anaconda)
-2. Edit the config.py & env variable (see in config.py)
-3. Run on localhost: python manage.py runserver
+INFOX 
 
 Language: Python3
 
 Framework: Flask
 
 Database: mongo
 
+Http server: uwsgi 
 
+Reverse Agent: nginx
 
-Code Overview:
+How to run:
 
-./config.py - Config for Flask
+1. Ramp up the environment according to environment.yaml (you can load by anaconda)
 
+ - install conda (package manager)
+    https://conda.io/docs/user-guide/install/download.html
+    Download Anaconda
 
+ - install dependencies (https://github.com/FancyCoder0/INFOX/blob/master/environment.yaml)
+   conda env create -f environment.yaml
 
-./config.ini - Config for using uWSGI
+ - install mongodb (install brew mongo)
```

Comment:
install brew mongo is wrong
correct is: brew install mongodb 
My idea is that just delete it.